ui/app/components/wizard/features-selection.js

```diff
@@ -44,13 +44,15 @@ export default Component.extend({
         steps: ['Enabling a secrets engine', 'Adding a secret'],
         selected: false,
         show: true,
+        permission: 'secrets',
       },
       {
         key: 'authentication',
         name: 'Authentication',
         steps: ['Enabling an auth method', 'Managing your auth method'],
         selected: false,
         show: true,
+        permission: 'access',
```

Comment:
This one is almost correct - we need to know if they can enable a method, but just `'access'` on the permissions doesn't do this.